swagger.yaml

```diff
@@ -4528,7 +8877,7 @@ definitions:
       cluster_network_cidr:
         type: string
         pattern: '^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)[\/]([1-9]|[1-2][0-9]|3[0-2]?)$'
-      cluster_network_host_prefix:
+      cluster_network_agent_prefix:
```

Comment:
This is causing issues we should have v1cluster_default_config with the old value 'cluster_network_host_prefix'

Reply:
fixed